ext/native-decls/MumbleDoesChannelExist.md

```diff
@@ -0,0 +1,14 @@
+---
+ns: CFX
+apiset: client
+---
+## MUMBLE_DOES_CHANNEL_EXIST
+
+```c
+void MUMBLE_DOES_CHANNEL_EXIST(int channel);
+```
+
+Check whether specified channel exists on the Mumble server.
+
+## Parameters
+* **channel**: A game voice channel ID.
```

Comment:
Add a return value description
```

## Return value
True if the specific channel exists. False otherwise.
```

Reply:
If nothing changed, natives never had description when name was obvious.

Reply:
Even when they were unneeded `## Return value` is usually added for every native declaration that will return something.